src/AvtaleContext.tsx

```diff
@@ -64,11 +65,15 @@ export interface Context {
         deltakerFnr: string,
         arbeidsgiverFnr: string
     ) => Promise<Avtale>;
+    hentRolle: (avtaleId: string) => void;
 }
 
+export type Rolle = 'DELTAKER' | 'ARBEIDSGIVER' | 'VEILEDER';
+
 // tslint:disable no-empty
 const AvtaleContext = React.createContext<Context>({
     avtale: tomAvtale,
+    rolle: 'DELTAKER',
```

Comment:
"Default" rolle burde kanskje være `null`? Vi har ikke lagt inn noen andre ferdige verdier i dette objektet

Reply:
Det er noe som skurrer med disse default-verdiene, som aldri egentlig skal brukes til noe. Må vi ha de?

Reply:
@LarsAndreasNav Er ikke det det samme som å ha en rolle `INGEN_ROLLE`?

Reply:
@mbolstad Nei, vi har ikke noe bruk for det. Det eneste er at React krever et eller annet default-objekt for å opprette en context. Kan erstattes med f.eks.
`const AvtaleContext = React.createContext<Context>({} as Context);`
Det gir samme effekt.